developer/docs/help/reference/kmc/cli/reference.md

```diff
@@ -294,6 +306,163 @@ For more information on the purpose of `analyze osk-char-use` and
 `analyze rewrite-osk-from-char-use`, see
 [`&displayMap`](/developer/language/reference/displaymap).
 
+## `kmc generate keyman-keyboard` options
+
+Generates a new keyboard project with .kmn format keyboard. An ID must be
+specified for the output of the project, and an output folder (`-o`). A new
+folder with the keyboard ID as its name will be created under the output folder,
+and the files in that folder will follow the
+[standard file layout](../../file-layout).
+
+`-t, --target <target>`
+
+: Target platforms for the project. Use the values from
+  [`&targets`](/developer/language/reference/targets). Multiple targets may be
+  specified, each prefixed with `-t`, or space-separated, surrounded by
+  quotation marks (e.g. `-t windows -t linux` or `-t "windows linux"`) (default:
+  `any`)
+
+`-o, --out-path <path>`
+
+: Specifies the parent folder for the project; the folder may already exist. The
+  project will be generated in a new folder named with the ID of the project
+  under this path, and that project folder must not exist.
+
+`-n, --name <name>`
+
+: Keyboard descriptive name, used in the
+  [`&name` store](/developer/language/reference/name) (default: the ID of the
+  project)
+
+`-c, --copyright <copyright-name>`
+
+: [`&copyright`](/developer/language/reference/copyright) holder for the
+  project. Do not include the '(C)' or '&copy;' prefixes (default: the author
+  of the keyboard)
+
+`-v, --version <version-string>`
+
+: [`&version`](/developer/language/reference/version) of the keyboard, (default:
+  "1.0").
+
+`-L, --language-tag <bcp-47 tag>`
```

Comment:
I just noticed.

Is lower-case `-l` already in use?

Reply:
Yes, `-l, --log-level` is a global parameter